### PR TITLE
Updates to the Viewport Interaction Model documentation

### PR DESCRIPTION
**Pull request by hultonha**
Following several recent PRs to https://github.com/o3de/o3de (notably https://github.com/o3de/o3de/pull/4263 and https://github.com/o3de/o3de/pull/4094) the O3DE documentation needs to be updated to reflect these changes.

I believe this covers the most important parts - please let me know if I might have missed something.

Thanks!

### CODE REVIEW

File: content/docs/user-guide/editor/viewport/viewport-interaction-model-reference-space-model.md [hunk 1]
```diff
@@ -7,31 +7,26 @@ weight: 200
...
-1. To switch to world space, press and hold **Shift**. To switch to local space, press and hold **Alt**. Use the manipulator to modify the entity.
+1. To switch to world space, press and hold **Shift**. Use the manipulator to modify the entity.
 
-{{< note >}}
-If you selected an entity that doesn't have a parent, your selection defaults to world space.
-Parent space is the transform that the parent entity has in the hierarchy. If an entity doesn't have a parent, the entity uses world space instead.
-{{< /note >}}
-
-In the following example, the entity moves in world space because the car does not have a parent. When you select the child entity (the tire), the manipulator switches to the parent space of that entity (the car).
+1. Viewport UI (W/P/L _World, Parent, Local_ in the top right of the viewport) allow you to pin a reference space.
```
Comment (micronAMZN) [inline content/docs/user-guide/editor/viewport/viewport-interaction-model-reference-space-model.md:29]: We have nifty little icons we can insert now.

```suggestion
1. Viewport UI icons ({{< icon "world.svg" >}} World, {{< icon "parent.svg" >}} Parent, and {{< icon "local.svg" >}} Local) in the top right of the viewport allow you to pin a reference space.
```

**Review by micronAMZN — CHANGES_REQUESTED** [review]

File: content/docs/user-guide/editor/viewport/_index.md [hunk 2]
```diff
@@ -1,32 +1,33 @@
...
+When you select an entity, you can specify whether to move the entity in its own reference frame, or in relation to another custom reference frame.
 
 ## Viewport Interaction Model Limitations 
 
 This feature has the following limitations:
-+ The Viewport Interaction Model supports component entities only and doesn't support legacy entities, such as terrain and vegetation.
-+ When you enable the Viewport Interaction Model, this feature removes some toolbar options in the O3DE Editor Interface, such as vertex snapping.
-+ You can't change or customize shortcuts for the Viewport Interaction Model.
+
++ You can't yet change or customize shortcuts for the Viewport Interaction Model.
```
[the remark this answers is not in the document]
Comment (hultonha) [reply content/docs/user-guide/editor/viewport/_index.md:30]: That's totally fair, I just really want us to fix it which is why I likely phrased it this way. It is going to happen in future but we can't say exactly when, I can just rephrase slightly.

File: content/docs/user-guide/editor/viewport/viewport-interaction-model-reference-space-model.md (hunk 1 above)
Comment (hultonha) [reply content/docs/user-guide/editor/viewport/viewport-interaction-model-reference-space-model.md:29]: Okay nice, do we have these icons already? Or will I need to find and add them?

Comment (hultonha) [reply content/docs/user-guide/editor/viewport/viewport-interaction-model-reference-space-model.md:29]: I found the icon and checked and it is actually white, so might display poorly on the page (it'll just be invisible to the reader). Is there a solution for this?

**Review by micronAMZN — APPROVED** [review]

**Review by micronAMZN — APPROVED** [review]
DCO is failing but the changes are good. Need to amend commits with sign-off.

**Comment by hultonha** [comment]
> 
> 
> DCO is failing but the changes are good. Need to amend commits with sign-off.

Argh that happened when I accepted your suggested changes ☹️ I'll see if I can fix-up locally and do a push with --force-with-lease

**Comment by hultonha** [comment]
The icons came out okay which is great!

image

**Comment by micronAMZN** [comment]
> 
> 
> > DCO is failing but the changes are good. Need to amend commits with sign-off.
> 
> Argh that happened when I accepted your suggested changes ☹️ I'll see if I can fix-up locally and do a push with --force-with-lease

Yep I think we've all made this mistake repeatedly. When you accept edit changes, you need to add the sign off line in the comment. If you are doing this through the web interface, you may need to use an anonymized GitHub email, which you can find in your GitHub email settings.

**Comment by hultonha** [comment]
> 
> 
> > > DCO is failing but the changes are good. Need to amend commits with sign-off.
> > 
> > 
> > Argh that happened when I accepted your suggested changes ☹️ I'll see if I can fix-up locally and do a push with --force-with-lease
> 
> Yep I think we've all made this mistake repeatedly. When you accept edit changes, you need to add the sign off line in the comment. If you are doing this through the web interface, you may need to use an anonymized GitHub email, which you can find in your GitHub email settings.

Good to know for future reference, they've all been updated now with signoffs, you might need to approve again I'm afraid 😬 Thanks for taking a look! 😄

**Comment by hultonha** [comment]
Just need one more approval, could you take a look please @JwGedit? Thanks!

File: content/docs/user-guide/editor/viewport/viewport-interaction-model-reference-space-model.md [hunk 3]
```diff
@@ -7,31 +7,26 @@ weight: 200
 You can use the Viewport Interaction Model to create reference spaces for your selected entities. This enables you to customize your selected entities in relation to a reference space.
 
 When working with reference spaces, remember the following rules:
-+ New selections always default to their parent space. This will be the world coordinate space if the entity does not have a parent.
++ New selections always default to local space.
 + Holding **Shift** aligns the manipulator to the world coordinate space.
-+ Holding **Alt** aligns the manipulator to an entity's local coordinate space.
-+ You can easily access custom spaces by moving the manipulator (press **Ctrl** and click and drag) or picking a target entity as a reference space (press **Ctrl** and **Alt** and click a target entity).
++ Holding **Alt** changes the manipulator to have _individual_ influence as opposed to _group_ influnce (the default).
++ You can easily access custom reference spaces by moving the manipulator (press and hold **Ctrl** and click and drag the manipulaotr) or picking a target entity as a reference space (press hold **Ctrl** and **Alt** and click a target entity).
```
Comment (JwGedit) [inline content/docs/user-guide/editor/viewport/viewport-interaction-model-reference-space-model.md:13]:
```suggestion
+ You can access custom reference spaces by moving the manipulator (press and hold **Ctrl** and then choose and drag the manipulator) or picking a target entity as a reference space (press hold **Ctrl** and **Alt** and click a target entity).
```

File: content/docs/user-guide/editor/viewport/viewport-interaction-model-reference-space-model.md [hunk 4]
```diff
@@ -7,31 +7,26 @@ weight: 200
...
 
-The Viewport Interaction Model simplifies the mental mode for you and doesn't require you to keep track manually of the last space (local or world) that you were using. You can specify a target and define that target as custom reference spaces. This new reference space model covers any arbitrary transformation.
+The Viewport Interaction Model simplifies the mental model for you and doesn't require you to keep track manually of the last space (local or world) that you were using. You can specify a target and define that target as custom reference spaces. This new reference space model covers any arbitrary transformation.
```
Comment (JwGedit) [inline content/docs/user-guide/editor/viewport/viewport-interaction-model-reference-space-model.md:15]:
```suggestion
The Viewport Interaction Model simplifies the mental model for you by not requiring you to manually track the last local or world space that you were using. You can specify a target and define that target as custom reference spaces. This new reference space model covers any arbitrary transformation.
```

**Review by JwGedit — CHANGES_REQUESTED** [review]

**Comment by hultonha** [comment]
Thank you for taking a look @JwGedit, all edits have been applied :+1:

**Comment by hultonha** [comment]
Can this please be merged now @JwGedit or @micronAMZN? Thanks!

**Review by JwGedit — APPROVED** [review]

**Comment by sptramer** [comment]
Two approvals, merging.

**Comment by hultonha** [comment]
Awesome thank you very much @sptramer! 🙂

**Pull request merged**